build_extra/rust/get_rust_ldflags.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+# Copyright 2018 Bert Belder <bertbelder@gmail.com>
```

Comment:
We should change to "Deno authors" soon. Maybe after we move out of github.com/ry

Please add a big comment at the top of this file explaining the situation... I think this is likely to be confusing to future readers.

Reply:
Yeah I'll add a big comment to rust.gni and a small one here. You can't say this file is sparsely commented...

Reply:
comment: done